site/content/en/docs/tasks/manage/restricting_job_creation.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Restricting job creation with Validating Admission Policy"
+date: 2024-06-28
+weight: 10
+description: >
+  Implementing Validating Admission Policy to prevent job creation without queue name.
+---
+
+This page shows you how to set up a [Validating Admission Policy](https://kubernetes.io/docs/reference/access-authn-authz/validating-admission-policy) to prevent the creation of jobs without a queue name within a namespace.
+
+## Before you begin
+
+Ensure the following conditions are met:
+
+- A Kubernetes cluster is running.
+- The kubectl command-line tool can communicate with your cluster.
+
+## Creating a Validating Admission Policy
+
+To verify a job contains a queue name, check that the `kueue.x-k8s.io/queue-name` label exists and has a value set using the [Common Expression Language (CEL)](https://github.com/google/cel-spec).
+
+Here is an example of a ValidatingAdmissionPolicy that rejects job creation without a queue name:
```

Comment:
I think users/admins are likely to have two use-cases for which examples would be handy:
1. how to support more Job types - should we duplicate ValidatingAdmissionPolicy & ValidatingAdmissionPolicyBinding, or there is a way to deduplicate?
2. how to keep some namespace for administrative purposes (not reject without QueueName).

I'm ok either with extending the example (say for Job and JobSet), or adding a new example (say for excluded administrative namespace)

Reply:
I've updated the example to make the policy namespace-scoped and restrict JobSet creation as well.

Reply:
Thanks, I think it looks better, I leave a couple of more comments to make the example less abstract.